frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverviewWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { OverviewItem } from '@console/shared';
+import MonitoringOverview from './MonitoringOverview';
+import { FirehoseResult, LoadingBox } from '@console/internal/components/utils';
+import { EventKind } from '@console/internal/module/k8s';
```

Comment:
nit: can be above relative imports

Reply:
linter doesn't seem to care anymore

Reply:
@christianvogt Linter seems to have issues for some time now... I think it was broken when we upgraded everything - It doesn't catch import order I don't think. Might just not catch `@` import order.